src/sentry/api/serializers/models/project.py

```diff
@@ -311,14 +311,15 @@ def get_attrs(self, item_list, user):
                     'dateFinished': date_finished
                 }
 
-        latest_release_list = bulk_fetch_project_latest_releases(item_list)
-        latest_releases = {
-            r.actual_project_id: d
-            for r, d in zip(latest_release_list, serialize(latest_release_list, user))
+        # We  just return the version key here so that we cut down on response
+        # size
+        latest_release_verions = {
+            release.actual_project_id: {'version': release.version}
```

Comment:
This is fine for now, but it might be worth creating an actual serializer for this that can just be reused. I can see value in this probably existing in other places.

Reply:
I feel like we'd include a few more fields in that serializer, rather than just the version. If I run into other cases where we want to reduce what we're returning I'll make a SimpleReleaseSerializer or something along those lines.